NameLayerMain/src/vg/civcraft/mc/namelayer/command/commands/SetDefaultGroup.java

```diff
@@ -44,8 +44,7 @@ public boolean execute(CommandSender sender, String[] args) {
 			return true;
 		}
 		
-		GroupPermission gPerm = gm.getPermissionforGroup(g);
-		if (!gPerm.isAccessible(pType, PermissionType.BLOCKS)){
+		if (!gm.hasAccess(g.getName(), uuid, PermissionType.getPermission("BLOCKS"))){
```

Comment:
Yeah, if someone sets a default group they can't reinforce on, Citadel will just complain endlessly on /ctf -- so it should be fine.